src/runtime.rs

```diff
@@ -7,9 +7,12 @@ use parking_lot::lock_api::{RawRwLock, RawRwLockRecursive};
 use parking_lot::{Mutex, RwLock};
 use rustc_hash::{FxHashMap, FxHasher};
 use smallvec::SmallVec;
-use std::hash::{BuildHasherDefault, Hash};
 use std::sync::atomic::{AtomicUsize, Ordering};
 use std::sync::Arc;
+use std::{
+    hash::{BuildHasherDefault, Hash},
+    panic::RefUnwindSafe,
```

Comment:
Let's keep import style consistent. Should probably fix this on rust-analyzer's side, to just guess the style if there isn't explicit config, cc @Veykril

Reply:
Guessing the style is an interesting thought, will note that down as an issue to remind me 👍